Tools/visualize_log.py

```diff
@@ -1,221 +1,211 @@
-import argparse
+"""Visualizes the telemetry data and events."""
+
 import glob
 import os
 import platform
 
 import matplotlib.pyplot as plt
+import mpl_config
 import numpy as np
 import pandas as pd
-from mpl_toolkits.mplot3d import Axes3D
+from absl import app, flags, logging
+
+FLAGS = flags.FLAGS
+
+# Telmetry file prefix.
+TELEMETRY_FILE_PREFIX = "sim_telemetry_"
 
+# Event log prefix.
+EVENT_LOG_FILE_PREFIX = "sim_events_"
 
-def get_logs_directory():
-    if platform.system() == "Windows":
+
+def _get_logs_directory() -> str:
+    """Returns the path to the logs directory."""
+    system = platform.system()
+    if system == "Windows":
         return os.path.expandvars(
-            r"%USERPROFILE%\AppData\LocalLow\BAMLAB\micromissiles\Telemetry\Logs"
-        )
-    elif platform.system() == "Darwin":
+            r"%USERPROFILE%\AppData\LocalLow\BAMLAB\micromissiles\Logs")
+    elif system == "Darwin":
         return os.path.expanduser(
-            "~/Library/Application Support/BAMLAB/micromissiles/Telemetry/Logs"
-        )
+            "~/Library/Application Support/BAMLAB/micromissiles/Logs")
+    elif system == "Linux":
+        return os.path.expanduser("~/.config/unity3d/Logs")
```

Comment:
I feel like this is wrong? Shouldn't it be

```suggestion
        return os.path.expanduser("~/.config/unity3d/BAMLAB/micromissiles/Logs")
```

Hunch tho

Reply:
Fixed in #160